### PR TITLE
gsocproposals: add Go-interpreter

### PR DESCRIPTION
**Pull request by sbinet**

### CODE REVIEW

File: _gsocproposals/proposal_GoHEPInterpreter.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Launching Wagon, a WebAssembly interpreter in Go
+layout: plain
+project: Go-HEP
+organization: CERN
```
Comment (jouvin) [inline _gsocproposals/proposal_GoHEPInterpreter.md]: @sbinet is the organization really CERN? You may want to create another one after #88 has been merged.

Comment (sbinet) [inline _gsocproposals/proposal_GoHEPInterpreter.md]: right.
should be Go-HEP, then. (I just followed the recommendation from https://github.com/HEP-SF/hep-sf.github.io/pull/83/files#r99058593)

Comment (sbinet) [inline _gsocproposals/proposal_GoHEPInterpreter.md]: done.

Comment (jouvin) [inline _gsocproposals/proposal_GoHEPInterpreter.md]: It is a fastly moving target!!! As soon as #90 is merged, it should be very easy to add an organization but the organization should be a real one (a legal entity) I guess. Adding @hegner who knows probably better the constraints.

Comment (hegner) [inline _gsocproposals/proposal_GoHEPInterpreter.md]: project is fine with Go-HEP I believe. Organization would be your home institute, which we'd need to add.

Comment (sbinet) [inline _gsocproposals/proposal_GoHEPInterpreter.md]: done.

**Comment by jouvin** [comment]
Wait for #93 to be merged and update of the organization before merging...

**Pull request merged**